frame.py

```diff
@@ -118,9 +129,13 @@ def check(sum, i, j):
                 self.reset()
 
         # fmt: off
-        _sum = self.board[index[0]][0] + self.board[index[0]][1] + self.board[index[0]][2]
+        _sum = self.board[index[0]][0] + \
+            self.board[index[0]][1] + \
+            self.board[index[0]][2]
         check(_sum, index[0], None)
-        _sum = self.board[0][index[1]] + self.board[1][index[1]] + self.board[2][index[1]]
+        _sum = self.board[0][index[1]] + \
+            self.board[1][index[1]] + \
+            self.board[2][index[1]]
```

Comment:
Make this [pep8](https://pep8.org/#break-before-or-after-binary-operator) compliant.